public/css/index.css

```diff
@@ -0,0 +1,102 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins&display=swap");
+
+* {
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  background: #1e1e25;
+}
+
+.form {
+  position: absolute;
+  left: 0;
+  right: 0;
+  background: #27293d;
+  height: 90%;
+  width: 100%;
```

Comment:
90% czego 100% czego staraj się unikać % jak nie masz zdefiniowanej jakiejś wartości nadrzędnej bo może to słabo wyglądać na różnych ekranach.